pkg/apis/config/feature_flags.go

```diff
@@ -148,13 +158,15 @@ func GetFeatureFlagsConfigName() string {
 
 // NewFeatureFlagsFromMap returns a Config given a map corresponding to a ConfigMap
 func NewFeatureFlagsFromMap(cfgMap map[string]string) (*FeatureFlags, error) {
+	cfgMapKeys := sets.StringKeySet(cfgMap)
```

Comment:
can we store all known key to a set and check if there are unknown keys?

Reply:
@Yongxuanzhang Oh I got it. That way could be better for future-proof and readability (specially for `DeprecatedOrIgnorableFeatureFlags`)